.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -364,7 +370,7 @@ jobs:
     env:
       DOCKER_IMAGE: ${{ needs.build.outputs.docker_image }}
       DOCS_TYPE: ${{ matrix.docs_type }}
-      WITH_PUSH: ${{ github.event_name == 'schedule' }}
+      WITH_PUSH: ${{ github.event_name == 'schedule' || startsWith(github.event.ref, "refs/tags/v1.") }}
```

Comment:
@atalman @seemethere do you guys think its ok to limit the conditions to `v1` tags, or extend it to all tags starting with `v`?(as we are not supposed to have any)

Reply:
@malfet We don't have any now, and maybe for some more time, since we are on 1.11 release. However when we decide to go to release v2.0 we will have these tags, don't we ?